azure_pipelines/stages/unity-build.yml

```diff
@@ -5,6 +5,7 @@ stages:
   jobs:
     - job:
       displayName: ${{ format('{0}', parameters.image) }}
+      timeoutInMinutes: 120
```

Comment:
Seems unrelated but I am fine with leaving it in.

Reply:
Yeah I kept hitting the default timeout which was 60 min do I doubled it.

Although I did find more errors with our build scripts I need to address. I might actually just pull those out.

Reply:
https://github.com/XRTK/XRTK-Core/pull/640